arangod/Transaction/Methods.cpp

```diff
@@ -2827,6 +2861,12 @@ futures::Future<OperationResult> Methods::countInternal(std::string const& colle
   return futures::makeFuture(countLocal(collectionName, type, options));
 }
 
+// perform a (deferred) intermediate commit if required
+Result Methods::performIntermediateCommitIfRequired(DataSourceId collectionId) {
+  [[maybe_unused]] bool hasPerformedIntermediateCommit = false;
+  return _state->performIntermediateCommitIfRequired(collectionId, hasPerformedIntermediateCommit);
```

Comment:
I think I found no caller that actually uses the value stored in `hasPerformedIntermediateCommit`. Can we get rid for that parameter?

Reply:
Good catch. Fixed!